cmd/eksctl-anywhere/cmd/internal/commands/artifacts/download.go

```diff
@@ -29,34 +29,41 @@ type Packager interface {
 }
 
 type Download struct {
-	Reader           Reader
-	Version          version.Info
-	ImageMover       ImageMover
-	ChartDownloader  ChartDownloader
-	Packager         Packager
-	TmpDowloadFolder string
-	DstFile          string
+	Reader                   Reader
+	Version                  version.Info
+	ImageDownloader          ImageMover
```

Comment:
nit: can we rename to `BundleImageDownloader` or something to distinguish from `EksaToolsImageDownloader`?